extensions-contrib/spectator-histogram/src/main/java/org/apache/druid/spectator/histogram/SpectatorHistogramIndexBasedComplexColumn.java

```diff
@@ -59,72 +53,11 @@ public Object getRowValue(int rowNum)
   @Override
   public int getLength()
   {
-    return index.size();
+    return -1;
```

Comment:
Why do we want to claim a length of -1 here?

Reply:
The method is specced as returning the serialized size of the column in bytes, or -1 if unknown. `index.size()` returns a row count, which doesn't match the specced behavior. The `SpectatorHistogramIndexed` doesn't seem to know its own serialized size, and the `getLength()` method doesn't seem to be used anywhere important, so I figured changing this to -1 was a good idea.

Reply:
If it's not important then -1 seems fine.
We can very likely compute (or estimate an upper bound) of the size of the column if it will optimize something elsewhere.